lib/core/network/api/eos_service.dart

```diff
@@ -1,19 +1,18 @@
 import 'dart:async';
```

Comment:
needs to be merged into main to fix the issue with return values and dio.Response vs. whatever Response object we used before.